integration_test/test_integration.py

```diff
@@ -88,10 +96,16 @@ def test_integration(tmp_path, capfd, ebuild):
     verify_files([(dist_file, pkg_info.checksum)])
     with tarfile.open(dist_file, "r") as tarf:
         for directory in pkg_info.directories:
-            for filename in ("Cargo.lock", "Cargo.toml"):
-                member = tarf.getmember(f"{directory}/{filename}")
+            member = tarf.getmember(f"{directory}/Cargo.toml")
+            tarf.extract(member, tmp_path, set_attrs=False)
+            for current in (Path(directory) / "Cargo.lock").parents:
```

Comment:
Unless I'm missing something,

```suggestion
            for current in (PurePath(directory) / "Cargo.lock").parents:
```

`Path` refers to real filesystem path while `PurePath` is abstract.

Reply:
Yes, you are right, I have changed it.